reference/render/checken.html

```diff
@@ -73,89 +73,84 @@ <h2 class="dso-steps-indicator">
+      <h3 class="dso-section-handle"><button type="button" aria-expanded="true">Plaats van het bouwwerk
         </button>
-      </div>
+      </h3>
       <div class="dso-section-body">
-        <form>
-          <fieldset>
-            <legend>Plaats van het bouwwerk</legend>
-            <div class="form-group dso-input dso-input-text">
-              <div class="dso-label-container">
-                <label for="form-vertical-omgevingsvergunningPostcode" class="control-label">
-                  Postcode
-                </label>
-              </div>
-              <div class="dso-field-container">
-                <input type="text" id="form-vertical-omgevingsvergunningPostcode" class="form-control" size="8">
-              </div>
-            </div>
-            <div class="form-group dso-input dso-input-text">
-              <div class="dso-label-container">
-                <label for="form-vertical-omgevingsvergunningHuisnummer" class="control-label">
-                  Huisnummer
-                </label>
-              </div>
-              <div class="dso-field-container">
-                <input type="text" id="form-vertical-omgevingsvergunningHuisnummer" class="form-control" size="8">
-              </div>
-            </div>
-            <div class="form-group dso-input dso-input-text">
-              <div class="dso-label-container">
-                <label for="form-vertical-omgevingsvergunningToevoeging" class="control-label">
-                  Huisnummer toevoeging
-                </label>
-              </div>
-              <div class="dso-field-container">
-                <input type="text" id="form-vertical-omgevingsvergunningToevoeging" class="form-control" size="8">
-              </div>
-            </div>
-            <div class="form-group dso-input dso-input-text">
-              <div class="dso-label-container">
-                <label for="form-vertical-omgevingsvergunningStraatnaam" class="control-label">
-                  Straatnaam
-                </label>
-              </div>
-              <div class="dso-field-container">
-                <input type="text" id="form-vertical-omgevingsvergunningStraatnaam" class="form-control">
-              </div>
-            </div>
-            <div class="form-group dso-input dso-input-text">
-              <div class="dso-label-container">
-                <label for="form-vertical-omgevingsvergunningWoonplaats" class="control-label">
-                  Woonplaats
-                </label>
-              </div>
-              <div class="dso-field-container">
-                <input type="text" id="form-vertical-omgevingsvergunningWoonplaats" class="form-control">
-              </div>
-            </div>
-          </fieldset>
-          <div class="dso-form-buttons">
-            <button type="submit" class="btn btn-primary"><span >Volgende stap</span></button>
+        <div class="form-group dso-input dso-input-text">
```

Comment:
Zie eerdere opmerking: het `<form>` dat hier verdwenen is, moet nog wel om de complete werkvorm heen.